src/components/heading.js

```diff
@@ -70,7 +70,7 @@ export default class Heading extends Component {
     return nextProps.fit !== prevState.fit ? { fit: nextProps.fit } : null;
   }
 
-  componentDidUpdate(prevState, prevProps) {
+  componentDidUpdate(prevProps) {
     if (prevProps.fit !== this.props.fit) {
```

Comment:
One thing to note here. By removing that first argument, you actually changed what `prevProps` means in the method. In the first iteration:

```
componentDidUpdate(prevState, prevProps) {
   if (prevProps.fit !== this.props.fit) {
      //
   }
}

`prevProps`, because of its position as the second argument, actually refers to `prevState`.

In the new iteration:

```
componentDidUpdate(prevProps) {
   if (prevProps.fit !== this.props.fit) {
      //
   }
}
```

`prevProps` correctly refers to `prevProps`, but your function hasn't changed at all to reflect that update. I think that's likely fine based on the other patterns you've implemented here, but want to make sure you understand the difference :)